src/patterns/task-list/index.md.njk

```diff
@@ -0,0 +1,75 @@
+---
+title: Task list pages
+description:
+section: Patterns
+theme: Pages
+aliases:
+layout: layout-pane.njk
+status: Experimental
+---
+
+Task list pages help users understand:
+
+- the tasks involved in completing a transaction
+- the order they should complete tasks in
+- when they have completed tasks
+
+![Task list example](task-list-whole.png)
+
+## When to use this pattern
+
+Only use a Task list page for longer transactions involving multiple tasks that users may need to complete over a number of sessions.
+
+Before you use a Task list, try to simplify the transaction. If you’re able to reduce the number of tasks or steps involved, you may not need a Task list page.
+
+## How it works
+
+You should show a Task list page:
+
+- at the start of the transaction
+- at the start of each returning session
+
+When using a Task list page in your service you need to:
+
+- group related actions into tasks
+- show the status of each task
+
+If there are lots of tasks to complete, you might also need to group them into sub-sections.
+
+### Show the status of the task
+
+Whenever you show a Task list page, make it clear to users which tasks they’ve completed by labelling them ‘Completed’.
+
+![Task list statuses example](task-list-statuses.png)
```

Comment:
Likewise, I think we could do with better alt text here.

Reply:
Fixed